Updated Simplified Chinese

SakuraiSakiiii opened the pull request:
## Translation Info
- **Author**: (`Saki`)
- **Translated Platform**: (`client`)
- **Translated Languages**: (`Simplified Chinese(zh_cn)`)

## Description
...
fixed some minor translation error,make it easier to read,added latest translation update  from original English Version
Added:Newest Demo Replay Translation
Removed:Older Translation Error
Fixed:Replaced Older Translation Error with Latest Translation,this type of translation is more popular in Chinese Minecraft Speedrun Community
currently I'm in Japan,cannot fix too much and I'm not a pure native Chinese Speaker too,I'll try my best to make contact with other Chinese player request for more translation fix(didn't mean I'm not a Chinese or so!)

the reason I start this PR is one of my friend DM me and says:maybe I can fix this

Status Update:
Updated Commit with some suggestion above,fixed some minor translation error,improved overall translation quality
compare to oldest one,basically it's a overhaul

from these days on I might be busy and will not have much time to keeping up the update,so I updated it to more latest version and basically it's a final version before newer one mod came out

Sorry Guys,But I need to focus on my school work until November,if when that time arrive mod just updated,I'll keep up if I go back to China or when I got free time,thank you and good night
see ya November,and be safe

SakuraiSakiiii committed:
Update zh_cn.editable.json

SakuraiSakiiii committed:
Update zh_cn.editable.json

SakuraiSakiiii committed:
Update zh_cn.editable.json

SakuraiSakiiii committed:
Update zh_cn.editable.json

SakuraiSakiiii committed:
Update zh_cn.editable.json

SakuraiSakiiii committed:
Update zh_cn.editable.json

SakuraiSakiiii committed:
Update zh_cn.editable.json

SakuraiSakiiii committed:
Update zh_cn.editable.json

SakuraiSakiiii committed:
Update zh_cn.editable.json

SakuraiSakiiii committed:
Update zh_cn.editable.json

FurryR commented:
# Requested change

- Conflict with #21 (Pending)

---

Please notice that the developers of `MCSR Ranked` are not Chinese, so you may need to describe your changes more specifically in order to ensure there are no misunderstandings.

Also, it might take a long time to wait your pull request being merged. Please keep your translation up-to-date so that you won't need to make another pull request. The original English translation file from GitHub is not always synchronized with the mod's translation, so you might need to extract the mod to get the latest translation file.

I will notify @RedLime later so that they can approve your pull request faster.

SakuraiSakiiii commented:
> # Requested change
> * Conflict with [📝 zh_CN translation update #21](https://github.com/MCSR-Ranked/translations/pull/21) (Pending)
> 
> Please notice that the developers of `MCSR Ranked` are not Chinese, so you may need to describe your changes more specifically in order to ensure there are no misunderstandings.
> 
> Also, it might take a long time to wait your pull request being merged. Please keep your translation up-to-date so that you won't need to make another pull request. The original English translation file from GitHub is not always synchronized with the mod's translation, so you might need to extract the mod to get the latest translation file.
> 
> I will notify @RedLime later so that they can approve your pull request faster.

sure,keep in contact  with me on discord or line if no problem

RedLime commented:
Don't worry about outdated strings. Current files are not outdated ;)

SakuraiSakiiii commented:
> ## Translation Info
> * **Author**: (`Saki`)
> * **Translated Platform**: (`client`)
> * **Translated Languages**: (`Simplified Chinese(zh_cn)`)
> 
> ## Description
> ... fixed some minor translation error,make it easier to read,added latest translation update from original English Version
Added:Newest Demo Replay Translation
Removed:Older Translation Error
Fixed:Replaced Older Translation Error with Latest Translation,this type of translation is more popular in Chinese Minecraft Speedrun Community
currently I'm in Japan,cannot fix too much and I'm not a native Chinese Speaker too,I'll try my best to make contact with other Chinese player request for more translation fix

SakuraiSakiiii commented:
> Don't worry about outdated strings. Current files are not outdated ;)

thanks,I just saw the newest update and forked it to do a new Translation,didn't think it will have a conflict with other pr,sry

FurryR commented:
Very appreciated! I am going to make a universal fix.

SakuraiSakiiii committed:
Update zh_cn.editable.json

SakuraiSakiiii commented:
> Very appreciated! I am going to make a universal fix.

keep in contact,can you leave a contact information,I mean any method you could use in China

SakuraiSakiiii committed:
Update zh_cn.editable.json

SakuraiSakiiii commented:
Status Update:
Updated Commit with some suggestion above,fixed some minor translation error,improved overall translation quality
compare to oldest one,basically it's a overhaul

SakuraiSakiiii commented:
> thank you for your efforts in translation, i have some modification suggestions here. before everything, i recommend replace all "你" to "您". then below are my suggestions for modifying some of the content.
> 
> line 10: "projectelo.text.socketmessage.ssl_verify_error": " SSL 握手失败。请尝试更新您的 Java 至最新版本", line 11: "projectelo.text.roommessage.none": "正在初始化房间...", line 13: "projectelo.text.roommessage.waiting": "等待请求被接受...", line 17 "projectelo.text.matchtype.casual.description": "与更大水平范围的对手进行休闲比赛。\n此模式不影响您的排位分。", line 19: "projectelo.text.matchtype.ranked.description": "在同时间同种子的情况下，\n与水平相近的玩家对战。\n\n通过和排位分相近的玩家对战\n来提高您的排名。", line 20: "projectelo.text.matchtype.private": "私人房间", line 22: "projectelo.text.matchtype.event.description": "在独特的规则下和随机玩家对战，\n此模式不影响您的排位分。", line 24: "projectelo.text.match_room": "比赛房间 - %s", line 25: "projectelo.text.match_code": "房间号: ", line 26: "projectelo.text.insert_match_code": "房间号...", line 29: "projectelo.text.failed_load_timeline": "这场比赛没有分段时间的数据。 ", line 31: "projectelo.text.failed_find_match.full": "房间已满。 ", line 33: "projectelo.text.failed_find_match.maintaining": "由于服务器正在维护，您无法寻找对局。请查看我们的 Discord 服务器以了解最新情况。", line 34: "projectelo.text.failed_find_match.already_playing": "您已经加入了对局。这有可能是一个BUG，请反馈这个bug。", line 35: "projectelo.text.failed_find_match.not_available_for_spectator": "您正在使用旁观者账户。您只能旁观私人对局。", line 36: "projectelo.text.insert_seed": "世界种子", line 37: "projectelo.text.insert_seed.description": "留空以使用随机种子", line 38: "projectelo.text.completion_percentage": "几位玩家完成时结束比赛", line 39: "projectelo.text.completion_percentage.description.1.1": "当有一名玩家完成", line 40: "projectelo.text.completion_percentage.description.1.2": "或只剩一名玩家未完成时结束比赛。", line 41: "projectelo.text.completion_percentage.description.2.1": "当 %s 名玩家完成", line 42: "projectelo.text.completion_percentage.description.2.2": "或当所有人都完成/放弃了时结束比赛。", line 43: "projectelo.text.completion_percentage.description.0": "不管此项如何，房主可以强制终止比赛。", line 47: "projectelo.text.room_name.description": "比赛期间在 "当前比赛" 位置显示自定义大厅名。大厅名的最大长度为 %s 个字符。您需要有 Tier 2 或更高的 Patreon 支持者等级才能使用此功能。", line 48: "projectelo.text.confirm_reset_world": "是否确定要重置当前世界？", line 49: "projectelo.text.confirm_leave_match": "是否确定要放弃本场比赛？", line 50: "projectelo.text.confirm_delete_replay": "是否确定要删除该回放？ (%s)", line 51: "projectelo.text.confirm_end_match": "是否确定要结束这场比赛？", line 67: "projectelo.text.started_seed_change_vote": "%s 发起了更换种子投票。如果您在 30 秒内同意该请求，比赛将会更换种子重新开始。", line 69: "projectelo.text.rejected_seed_change_vote": "对手拒绝了更换种子。", line 71: "projectelo.text.rejected_draw_vote": "对手拒绝了平局请求。", line 72: "projectelo.text.disconnected_server.normal": "与 MCSR Ranked 服务器断开连接。", line 73: "projectelo.text.disconnected_server.reconnect": "与服务器断开连接，正在尝试重连... (剩余 %s 次尝试)", line 75: "projectelo.text.disconnected_server.success": "成功重新连接到 MCSR Ranked 服务器。", line 77: "projectelo.text.not_played_match": "您还没玩过排位或者匹配模式。", line 79: "projectelo.text.manually_update": "请手动更新。(自行安装最新版模组) ", line 82: "projectelo.text.afk_warning": "警告！多场比赛中挂机可能会导致您的账户被限制使用。 ", line 86: "projectelo.text.profile.latest_time": "最后上线时间", line 94: "projectelo.text.profile.ranked_statistics": "排位数据", line 97: "projectelo.text.profile.best_elo_rate": "排位分 (最佳 / 上个赛季)", line 99: "projectelo.text.profile.muted": "已屏蔽", line 104: "projectelo.text.match.decay": "自动掉分", line 111: "projectelo.text.achievement_badges": "成就奖章", line 131: "projectelo.text.not_enough_space_replay": "硬盘空间不足，无法保存回放录像。请至少预留 100 MB 可用空间。", line 136: "projectelo.info.ranks.unrated": "未定位的玩家需要至少完成 5 场比赛才能确定段位。", line 138: "projectelo.info.match_format.queue_range": "队列范围", line 139: "projectelo.info.match_format.decay_elo": "排行榜上的前 %s 名玩家若超过 %s 天没有游玩排位模式，那么会每日自动扣除 %s 排位分，直至游玩为止。", line 144: "projectelo.title.match_room": "比赛房间", line 172: "projectelo.timeline.blind_travel": "%s 在下界坐标 %s, %s, %s 处盲传", line 173: "projectelo.timeline.blind_travel.compact": "%s 在 %s, %s, %s 盲传", line 189: "projectelo.match_flag.igt_mode.description": "计时将使用游戏内时间 (IGT) 而不是现实时间 (RTA)。如果禁用了 '等待全员完成' 的设置，最先完成目标的玩家仍然会获胜。", line 192: "projectelo.match_flag.show_first_advancement_only": "每个进度只显示一次", line 193: "projectelo.match_flag.show_first_advancement_only.description": "如果有玩家已经完成了某进度，再次完成此进度时将不会显示", line 196: "projectelo.match_flag.waiting_all_players": "等待所有玩家完成", line 197: "projectelo.match_flag.waiting_all_players.description": "在所有玩家完成前，比赛不会结束。", line 209: "projectelo.match_flag.disable_copy_location.description": "在比赛期间禁用 F3 + C 以防止复制坐标。相当于禁用要塞计算器。", line 219: "projectelo.match_flag.force_surface_blind.description": "第一次盲传的传送门的y坐标将高于在地狱传送时的y坐标。", line 221: "projectelo.match_flag.no_logs.description": "比赛数据将不会被保存在服务器上，并且筛选种子将不可用。玩家上限将提升至 300。(请参照观察者账户的文档)", line 227: "projectelo.player.status.blinded": "正在寻找要塞", line 228: "projectelo.player.status.eye_spy": "已找到要塞", line 243: "projectelo.tooltip.copy_match_code": "点击复制房间号。", line 245: "projectelo.tooltip.toggle_match_code_visibility": "点击显示/隐藏房间号。", line 251: "projectelo_tooltip.elo_decay": "由于您长时间未进行排位赛，您的排位分已减少", line 254: "projectelo.toast.copied_match_code": "房间号成功复制到剪贴板。", line 262: "projectelo.toast.blocked_feature_private": "在私人房间中不能使用此功能", line 263: "projectelo.toast.blocked_feature_private_host": "在这个私人房间中只有拥有者能使用这个功能", line 274: "projectelo.button.create_room": "创建房间", line 275: "projectelo.button.join_room": "加入房间", line 276: "projectelo.button.leave_room": "离开房间", line 314: "projectelo.button.load_all_from_match": "加载上次比赛的设置", line 333: "projectelo.button.elo_leaderboard": "排位分排行榜", line 357: "projectelo.option.hide_profile.description": "在排位赛中，不显示您的资料。", line 359: "projectelo.option.hide_opponent_profile.description": "在排位赛中，不显示对手的资料。", line 368: "projectelo.option.hide_elo_score": "比赛时的排位分", line 369: "projectelo.option.hide_elo_score.description": "选择是否在比赛中显示你/对手的排位分。", line 386: "projectelo.tip_message.0": "您知道吗？ "MCSR" 意思是我的世界(MC)速通(SR)。", line 387: "projectelo.tip_message.1": "记得加入我们的 Discord 频道！", line 388: "projectelo.tip_message.2": "在一场比赛中，所有玩家都会有一样的随机数...除了末影龙。", line 390: "projectelo.tip_message.4": "试试看不使用计算器进行速通。", line 393: "projectelo.tip_message.7": "如果您刚开始玩排位模式，先进行一些练习会很有帮助", line 394: "projectelo.tip_message.8": "在排行榜上没有找到自己？请绑定您的 Discord 账户", line 395: "projectelo.tip_message.9": "当玩家靠近返回传送门时，末影龙将有 50% 的概率向玩家发射火球，但当离得远时概率则下降到 10%左右。", line 400: "projectelo.tip_message.14": "要塞在生成时会避免生成在海洋和沼泽群系，有时会被推向旁边的生物群系。", line 407: "projectelo.tip_message.21": "船可以用来防止摔落伤害，除非你刚好下落了 12, 13, 49, 51, 111, 114, 198, 202, 310, 或者 315 格。", ilne 420: "projectelo.tip_message.34": "即使失去了打开箱子玩家的视野，猪灵仍然会愤怒 1-2 秒。", line 421: "projectelo.tip_message.35": "制作光灵箭需要很多萤石，但可以使你箭的数量翻倍。", line 426: "projectelo.tip_message.40": "传送门房间不像其它要塞房间一样会被自然生成的地形 (比如峡谷) 所覆盖。", line 427: "projectelo.tip_message.41": "每个末地主岛将生成相同的 10 根黑曜石柱，只是布局不同。", line 429: "projectelo.tip_message.43": "无论伤害来源于谁，只要末地水晶被摧毁，那么末影龙就一定会向玩家发射火球。", line 430: "projectelo.tip_message.44": "堡垒遗迹里的箱子将至少包含 3 个铁锭和 5 个黑曜石。", line 438: "projectelo.text.queue_in_background_cancel": "按下 Ctrl+Z 取消排队", line 444: "projectelo.button.download_replay.description": "您可以下载近期比赛的回放", line 445: "projectelo.button.queue_in_background": "在后台排队"

updated

miasamnia commented on client/zh_cn.editable.json:
```diff
@@ -417,19 +417,19 @@
-    "projectelo.tip_message.40": "传送门房间不像其它要塞房间一样，它并不会被自然生成的地形 (比如峡谷) 所覆盖。",
-    "projectelo.tip_message.41": "每个末影岛将生成相同的 10 根黑曜石柱，只是布局不同。",
+    "projectelo.tip_message.40": "传送门房间不像其它要塞房间一样会被自然生成的地形 (比如峡谷) 所覆盖。",
+    "projectelo.tip_message.41": "每个末影岛将生成相同的 10 根同样的黑曜石柱，只是布局不同。",
```
“末地主岛”maybe better than“末影岛”？

miasamnia replied:
and i think there are still some controversies, like“排名分","排名赛模式”, instead of"排位分","排位模式"

miasamnia replied:
and ”大厅“ , ”房间“ i'd say most of players call a room”房间“ instead of ”大厅“

SakuraiSakiiii replied:
sorry,but these translation type is I learn from my other friends,i will change the "末影岛" to more accurate "末影主岛",but I will keep other translation like "Lobby/大厅" and "Ranked/排名赛" for a while,all translation changes will depends on player feedback,thank you

miasamnia replied:
yeah i respect that, but still, it's ”末地主岛“ not "末影" . And as far as i can see, there's "room" in ranked and i didn't see any "lobby", maybe i'm wrong here but... i'm not sure.

SakuraiSakiiii replied:
fixed the typo,my bad,but the room or lobby,I do prefer lobby a little more,as far as I know and some streamer record,they all call it Party or Lobby,also the en_us file use the word that close to Lobby or Party too,I'll wait for your feedback,if more players prefers to Room/房间,I will change it after you reply,if I'm not busy

miasamnia replied:
yeah i compared yours with en_us file in DO_NOT_CHANGE directory.(i didn't know if it's outdated), and all there are just "room"... so which en_us file did you follow?

SakuraiSakiiii replied:
committed ;)

miasamnia replied:
I'm glad we reached a consensus  :3

SakuraiSakiiii replied:
in fact I use Lobby more is I saw from other players,at least to me is:join my lobby,at some point this are right ya'know,as a game party owner it doesn't have any problem to call it a "lobby",but it depends on player base,Chinese Community is apparently larger than others

miasamnia replied:
> in fact I use Lobby more is I saw from other players,at least to me is:join my lobby,at some point this are right ya'know,as a game party owner it doesn't have any problem to call it a "lobby",but it depends on player base,Chinese Community is apparently larger than others

yeah this is one of fun things in translation, you use a word in English but at the same situation, you don't use the literal translation when using another language, because there are already suitable words to use there.

SakuraiSakiiii replied:
I wonder what kind of communicate software use most in China,see maybe I can make one

miasamnia replied:
Tencent qq mostly

SakuraiSakiiii replied:
see if I can create one,maybe communicate will much easier in future

SakuraiSakiiii replied:
3656487283 thank you,no region lock

SakuraiSakiiii committed:
Update zh_cn.editable.json

SakuraiSakiiii committed:
Update zh_cn.editable.json

RedLime commented:
any approves?

SakuraiSakiiii commented:
> any approves?

sure!just wait for me to did the one last commit

miasamnia commented:
> any approves?

waiting @FurryR to make a final version i think. not sure

miasamnia commented:
> > any approves?
> 
> sure!just wait for me to did the one last commit

oh ok

SakuraiSakiiii committed:
Update zh_cn.editable.json

SakuraiSakiiii commented:
> any approves?

done with final commit!you can use it now!

RedLime commented:
alright I will just wait for @FurryR's comment :)

FurryR commented:
> alright I will just wait for @FurryR's comment :)

# Sorry

Hi I'm here! Sorry for replying so late. My social account was temporary banned for 7 days, lately.

# My opinion

So there are 3 versions of zh-cn translation and 2 pull requests about it. We have to resolve merge conflicts before this pr being merge. That means we have to close a pr and merge the other one.
Merging 2 prs & 3 versions may take some time, so we can't just simply merge this pr right now.
In addition, I haven't done a final verification yet.

# What to do

@SakuraiSakiiii Please add me as a collaborator so I can perform some little fixes & merge.
@RedLime Maybe we can create a `lang` directory in MCSR Ranked's home directory to load custom translations. With that we needn't crack the jar file to load translations. It violates EULA.

SakuraiSakiiii commented:
> > alright I will just wait for @FurryR's comment :)
> 
> # Sorry
> Hi I'm here! Sorry for replying so late. My social account was temporary banned for 7 days, lately.
> 
> # My opinion
> So there are 3 versions of zh-cn translation and 2 pull requests about it. We have to resolve merge conflicts before merging this pr. That means we have to close a pr and merge the other one. Merging 2 prs & 3 versions may take some time, so we can't just simply merge this pr right now. In addition, I haven't done a final verification yet.
> 
> # What to do
> @SakuraiSakiiii Please add me as a collaborator so I can perform some little fixes & merge. @RedLime Maybe we can create a `lang` directory in MCSR Ranked's home directory to load custom translations. With that we needn't crack the jar file to load translations. It violates EULA.

fixing issue:I hand over to other person who have time to make update
others:this translations is a general overhaul,it basically became a translation from scratch,I have no personal interest on closing mine,but it all depends on devs

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -1,231 +1,231 @@
-    "projectelo.text.roommessage.none": "初始化房间中...",
-    "projectelo.text.roommessage.finding": "寻找房间中...",
-    "projectelo.text.roommessage.waiting": "请等待房主处理加入请求。",
+    "projectelo.text.socketmessage.ssl_verify_error": "SSL 握手失败。请尝试更新您的 Java 版本至最新版",
```
```suggestion
    "projectelo.text.socketmessage.ssl_verify_error": "SSL 握手失败。请尝试更新您的 Java 版本至最新版。",
```

zh-cn: 缺少句号

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -1,231 +1,231 @@
-    "projectelo.text.open_replay_folder": "打开回放文件夹",
-    "projectelo.text.archived": "已封存",
+    "projectelo.text.open_replay_folder": "打开回放存放文件夹",
+    "projectelo.text.archived": "已打包",
```
```suggestion
    "projectelo.text.archived": "已固定",
```
zh-cn: 问题：”打包“指代不明。MCSR 的 archive 实际上是指让某些回放不会随着达到上限而自动删除。

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -1,231 +1,231 @@
+    "projectelo.text.archived": "已打包",
     "projectelo.text.replay_load_fail": "因错误而无法加载回放文件",
-    "projectelo.text.not_archived_replay_warning": "若不封存回放，更旧的回放将在创建新回放时自动删除。",
+    "projectelo.text.not_archived_replay_warning": "如果您不对您的回放进行打包，更旧的回放将在创建新回放时自动删除。",
```
```suggestion
    "projectelo.text.not_archived_replay_warning": "如果您不对您的回放进行固定，更旧的回放将在创建新回放时自动删除。",
```

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -284,34 +284,34 @@
-    "projectelo.button.archive_replay": "封存回放",
-    "projectelo.button.unarchive_replay": "解封回放",
+    "projectelo.button.filter": "筛选",
+    "projectelo.button.filter_archived_only": "仅显示打包的回放",
```
```suggestion
    "projectelo.button.filter_archived_only": "仅显示固定的回放",
```

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -284,34 +284,34 @@
-    "projectelo.button.unarchive_replay": "解封回放",
+    "projectelo.button.filter": "筛选",
+    "projectelo.button.filter_archived_only": "仅显示打包的回放",
+    "projectelo.button.archive_replay": "打包回放",
```
```suggestion
    "projectelo.button.archive_replay": "固定回放",
```

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -284,34 +284,34 @@
+    "projectelo.button.filter": "筛选",
+    "projectelo.button.filter_archived_only": "仅显示打包的回放",
+    "projectelo.button.archive_replay": "打包回放",
+    "projectelo.button.unarchive_replay": "解压回放",
```
```suggestion
    "projectelo.button.unarchive_replay": "取消固定回放",
```

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -284,34 +284,34 @@
     "projectelo.button.unlink_third_party": "解绑 %s",
     "projectelo.button.public_third_party": "公开",
-    "projectelo.button.public_third_party.description": "决定是否要在你的个人资料上显示你的 %s 标签/名字。",
+    "projectelo.button.public_third_party.description": "是否要在您的个人资料上显示你的 %s 标签/名字。",
```
```suggestion
    "projectelo.button.public_third_party.description": "决定是否要在您的个人资料上显示你的 %s 标签/名字。",
```
zh-cn: 问题；这句感觉有点生硬，“是否”让人感觉像是疑问句，然而句子末尾却是句号。我的建议是 revert。

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -284,34 +284,34 @@
     "projectelo.button.report": "举报",
-    "projectelo.button.report_toxic_chat": "嘴臭",
-    "projectelo.button.report_hacking": "开挂",
+    "projectelo.button.report_toxic_chat": "人身攻击",
```
```suggestion
    "projectelo.button.report_toxic_chat": "发言不当",
```

zh-cn: toxic chat 并不完全指人身攻击。

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -340,107 +340,107 @@
     "projectelo.option.scale_hud": "GUI 缩放",
     "projectelo.option.show_my_progress": "显示个人进度",
-    "projectelo.option.show_my_progress.description": "启用比赛现况 HUD 将显示你的进度。",
+    "projectelo.option.show_my_progress.description": "切换比赛实时 HUD 是否显示您的进度。",
```
zh-cn: 问题：这句感觉有点生硬。

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -340,107 +340,107 @@
-    "projectelo.option.notify_background_chat": "在失去焦点时提醒消息",
-    "projectelo.option.notify_background_chat.description": "当你的 Minecraft 实例未获取焦点时，如果有任何新的消息将发出提示音。",
-    "projectelo.warning.leaderboard_discord_link": "若要将排名显示在你的个人资料和排行榜上，必须将账户与 Discord 绑定。",
+    "projectelo.option.pre_generate_all_dimensions.description": "当启用时，所有的维度都将在比赛开始时预先生成。这会增加世界生成的时间，但是会加快进入维度的速度。\n(来自 Quesia)",
```
```suggestion
    "projectelo.option.pre_generate_all_dimensions.description": "当启用时，所有的维度都将在比赛开始时预先生成。这会使世界生成更慢，但是会加快进入维度的速度。\n(来自 Quesia)",
```

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -340,107 +340,107 @@
-    "projectelo.warning.leaderboard_discord_link": "若要将排名显示在你的个人资料和排行榜上，必须将账户与 Discord 绑定。",
+    "projectelo.option.pre_generate_all_dimensions.description": "当启用时，所有的维度都将在比赛开始时预先生成。这会增加世界生成的时间，但是会加快进入维度的速度。\n(来自 Quesia)",
+    "projectelo.option.notify_background_chat": "在后台提醒聊天消息",
+    "projectelo.option.notify_background_chat.description": "当您的 Minecraft 不在Focus模式下时，收到任意消息都将会提醒您。",
```
```suggestion
    "projectelo.option.notify_background_chat.description": "当您的 Minecraft 不在前台时，收到任意消息都将会提醒您。",
```

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -340,107 +340,107 @@
-    "projectelo.tip_message.39": "传送门房间只能在离要塞初始楼梯的 7 个房间外生成。",
-    "projectelo.tip_message.40": "传送门房间不像其它要塞房间一样，它并不会被自然生成的地形 (比如峡谷) 所覆盖。",
-    "projectelo.tip_message.41": "每个末影岛将生成相同的 10 根黑曜石柱，只是布局不同。",
+    "projectelo.tip_message.37": "对着一个已经被填充的末地传送门框架上右键末影之眼，它也不会被丢出去。",
```
```suggestion
    "projectelo.tip_message.37": "即使对着一个已经被填充的末地传送门框架上右键末影之眼，它也不会被丢出去。",
```

zh-cn: 这里是弄错了么？

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -340,107 +340,107 @@
     "projectelo.tip_message.42": "末影龙在特定间隔栖息的概率为 1 / 13，但当 10 个末影水晶均被破坏时则为 1 / 3。",
-    "projectelo.tip_message.43": "如果末地水晶被破坏，末影龙将会向玩家发射火球，不管伤害源来自哪里。",
-    "projectelo.tip_message.44": "预定的堡垒遗迹附近的箱子将至少包含 3 个以上的铁锭和 5 个以上的黑曜石。",
+    "projectelo.tip_message.43": "无论伤害来源是谁，只要末地水晶被摧毁，那么末影龙就一定会向玩家发射火球。",
```
```suggestion
    "projectelo.tip_message.43": "无论伤害来源是谁，只要末地水晶被摧毁，末影龙就一定会向玩家发射火球。",
```

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -340,107 +340,107 @@
+    "projectelo.tip_message.46": "下界要塞将一定在堡垒遗迹附近的 17 个区块内生成。您可以用饼图找到下界要塞的精准坐标。",
+    "projectelo.tip_message.47": "在 MCSR Ranked 中，有五个根据段位分来定的段位：煤炭、铁锭、黄金、绿宝石、以及钻石。每个段位都有 3 个等级，3 级是最高的。",
+    "projectelo.toast.replay_change_success": "成功修改了该回放",
+    "projectelo.toast.replay_change_fail": "回放修改失败！请检查您最新的log文件！",
```
```suggestion
    "projectelo.toast.replay_change_fail": "回放修改失败！请检查您最新的日志文件！",
```

FurryR commented on client/zh_cn.editable.json:
```diff
@@ -340,107 +340,107 @@
+    "projectelo.button.check_match_info": "打开比赛详细数据",
+    "projectelo.button.download_replay": "下载",
+    "projectelo.button.download_replay.description": "您可以在 最近比赛 中下载您的比赛回放文件",
+    "projectelo.button.queue_in_background": "后台队列（不影响您的交互）"
```
```suggestion
    "projectelo.button.queue_in_background": "后台排队（不影响您的操作）"
```

FurryR reviewed (changes requested):
Reviewed, nice pr but need some little changes 👍

notlin4 commented on client/zh_cn.editable.json:
```diff
@@ -340,107 +340,107 @@
+    "projectelo.tip_message.39": "传送门房间只能在离要塞初始楼梯的至少 7 个房间外生成。",
+    "projectelo.tip_message.40": "传送门房间不像其它要塞房间一样会被自然生成的地形 (比如峡谷) 所覆盖。",
+    "projectelo.tip_message.41": "每个末地主岛将生成相同的 10 根同样的黑曜石柱，只是布局不同。",
     "projectelo.tip_message.42": "末影龙在特定间隔栖息的概率为 1 / 13，但当 10 个末影水晶均被破坏时则为 1 / 3。",
```
```suggestion
    "projectelo.tip_message.42": "末影龙在特定间隔栖息的概率为1/13，但当10个末影水晶均被破坏时则为1/3。",
```

notlin4 commented on client/zh_cn.editable.json:
```diff
@@ -340,107 +340,107 @@
-    "projectelo.tip_message.43": "如果末地水晶被破坏，末影龙将会向玩家发射火球，不管伤害源来自哪里。",
-    "projectelo.tip_message.44": "预定的堡垒遗迹附近的箱子将至少包含 3 个以上的铁锭和 5 个以上的黑曜石。",
+    "projectelo.tip_message.43": "无论伤害来源是谁，只要末地水晶被摧毁，那么末影龙就一定会向玩家发射火球。",
+    "projectelo.tip_message.44": "堡垒遗迹里的箱子将至少包含 3 个铁锭和 5 个黑曜石。",
```
```suggestion
    "projectelo.tip_message.44": "堡垒遗迹里的箱子将至少包含3个铁锭和5个黑曜石。",
```

notlin4 commented on client/zh_cn.editable.json:
```diff
@@ -340,107 +340,107 @@
-    "projectelo.button.queue_in_background": "Queue in Background"
-}
\ No newline at end of file
+    "projectelo.tip_message.46": "下界要塞将一定在堡垒遗迹附近的 17 个区块内生成。您可以用饼图找到下界要塞的精准坐标。",
```
```suggestion
    "projectelo.tip_message.46": "下界要塞将一定在堡垒遗迹附近的17个区块内生成。您可以用饼图找到下界要塞的精准坐标。",
```

notlin4 commented on client/zh_cn.editable.json:
```diff
@@ -340,107 +340,107 @@
-}
\ No newline at end of file
+    "projectelo.tip_message.46": "下界要塞将一定在堡垒遗迹附近的 17 个区块内生成。您可以用饼图找到下界要塞的精准坐标。",
+    "projectelo.tip_message.47": "在 MCSR Ranked 中，有五个根据段位分来定的段位：煤炭、铁锭、黄金、绿宝石、以及钻石。每个段位都有 3 个等级，3 级是最高的。",
```
```suggestion
    "projectelo.tip_message.47": "在 MCSR Ranked 中，有五个根据段位分来定的段位：煤炭、铁锭、黄金、绿宝石、以及钻石。每个段位都有3个等级，3级是最高的。",
```

notlin4 commented on client/zh_cn.editable.json:
```diff
@@ -340,107 +340,107 @@
+    "projectelo.toast.replay_change_fail": "回放修改失败！请检查您最新的log文件！",
+    "projectelo.text.search_replay": "搜索回放",
+    "projectelo.text.downloading_replay_failed": "回放数据下载失败",
+    "projectelo.text.queue_in_background_cancel": "按下 Ctrl+Z 即可取消排队",
```
```suggestion
    "projectelo.text.queue_in_background_cancel": "按下Ctrl+Z即可取消排队",
```

notlin4 commented on client/zh_cn.editable.json:
```diff
@@ -340,107 +340,107 @@
+    "projectelo.toast.replay_download_success": "成功下载回放文件",
+    "projectelo.button.check_match_info": "打开比赛详细数据",
+    "projectelo.button.download_replay": "下载",
+    "projectelo.button.download_replay.description": "您可以在 最近比赛 中下载您的比赛回放文件",
```
```suggestion
    "projectelo.button.download_replay.description": "您可以在“最近比赛”中下载您的比赛回放文件",
```

notlin4 commented on client/zh_cn.editable.json:
```diff
@@ -1,231 +1,231 @@
-    "projectelo.text.recommend_auto_toggle_aa_any": "建议关闭 SpeedRunIGT 的 '自动切换到 Any%%/AA' 功能，否则计时器可能不会停止。",
-    "projectelo.text.waiting_players": "等待队列: %s",
-    "projectelo.text.playing_players": "游玩中: %s",
+    "projectelo.text.gui_position.move": "按 §eCtrl + 方向键 §f 来微调整您的位置。",
```
```suggestion
    "projectelo.text.gui_position.move": "按 §eCtrl + 方向键§f 来微调整您的位置。",
```

FurryR commented:
Uhh... Why is this PR not merged / closed yet?
